tests/e2e/core-tests/specs/index.js

```diff
@@ -69,6 +70,7 @@ const runMerchantTests = () => {
 	runProductEditDetailsTest();
 	runProductSearchTest();
 	runMerchantOrdersCustomerPaymentPage();
+	runMerchantOrderEmailsTest();
```

Comment:
The issue with adding this to `runMerchantTests` is that projects using the released version of the function will have failing tests when they update to the next version due to not having the mail plugin installed. Can we remove it from here and rename `test-order-emails.js` to order-emails.test.js?

Reply:
Oh that is a good call out--thanks for bringing it up! Those changes have been made.